### DISCUSSION TITLE
Can we adjust the clustering resolution similarly to scRNAseq?

### DISCUSSION THREAD

**Discussion by coopershawna** [stuart-lab/signac#528] [Q&A] [answered]
My current run separates spatially into 3 groups and 5 clusters. It is pretty apparent that one of my clusters should really be at least 2 based on marker genes.  In Seurat (RNAseq), it is pretty easy to just change the resolution and increase the clusters. How do I do this in Signac for ATAC clusters?

Is that part of the "algorithm = 3" from the human pbmc vignette?

Thank you!

**Comment by timoast** [comment, marked as answer]
The same function is used in Signac/Seurat for clustering (`FindClusters()`), so you can use the same parameters to adjust the clustering resolution (`resolution` parameter).

**Reply by coopershawna** [reply]
Fantastic! Thank you.
